api/token.ts

```diff
@@ -3,11 +3,13 @@ import { PluggyClient } from 'pluggy-sdk';
 
 const { PLUGGY_CLIENT_ID, PLUGGY_CLIENT_SECRET } = process.env;
 
-export default function (_: NowRequest, res: NowResponse) {
+export default function (req: NowRequest, res: NowResponse) {
   const pluggyClient = new PluggyClient({
     clientId: PLUGGY_CLIENT_ID,
     clientSecret: PLUGGY_CLIENT_SECRET,
   });
 
-  pluggyClient.createConnectToken().then(data => res.json(data));
+  const { itemId, options } = req.body || {}
```

Comment:
Shouldn't we validate the `options` object?

Reply:
Its a passthrough, validations will occur on the endpoint.

Reply:
My only concern would be what if the options object is empty or undefined, does it work with no errors? If it does, then I'm ok with it